travis_script_1.sh

```diff
@@ -0,0 +1,2 @@
+#!/usr/bin/env bash
+echo heloo
```

Comment:
Thank you. Please add later a line that explains the purpose of the dummy script.